server/api/organisation/organisation.validation.js

```diff
@@ -0,0 +1,24 @@
+const websiteRegex = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w.-]+)+[\w\-._~:/?#[\]@!$&'()*+,;=.]+$/
```

Comment:
Could we also extract this `regex` and share a common one between this new file and `person.validation.js`?

Reply:
Oh sweet that makes sense to take it out !!!